kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KieServerTest.java

```diff
@@ -255,9 +257,12 @@ public void testPrometheusEndpoint() {
 
     @Test
     public void testCommandServiceSetup() {
+        List<String> extensionsWithoutCommandSupport = asList(
+                PrometheusKieServerExtension.EXTENSION_NAME,
+                ScenarioSimulationKieServerExtension.EXTENSION_NAME);
         for (KieServerExtension extension : ((KieServerImpl) kieServer).getServerExtensions()) {
             KieContainerCommandService<?> tmp = extension.getAppComponents(KieContainerCommandService.class);
-            if (PrometheusKieServerExtension.EXTENSION_NAME.equals(extension.getExtensionName())) {
+            if (extensionsWithoutCommandSupport.contains(extension.getExtensionName())) {
                 assertNull(tmp);
             } else {
                 assertNotNull(tmp);
```

Comment:
Extension getExtensionName could be null in this case it occasionally will propogate to this case, is it intended to be so?

Reply:
To be honest this should not happen because `extension.getExtensionName()` is used as key of the extension. Btw added an assertion in the test to ensure it.

Reply:
Thanks